x-pack/test/security_functional/plugins/test_endpoints/server/init_routes.ts

```diff
@@ -177,6 +177,18 @@ export function initRoutes(
     }
   }
 
+  router.post(
+    {
+      path: '/session/_run_cleanup',
+      validate: {},
```

Comment:
__optional nit:__ 
```suggestion
      validate: false,
```

Reply:
Nice! Fixed in https://github.com/elastic/kibana/pull/148985/commits/797660275af473c825836320b4007530faed244f.